src/Radicle/Internal/CLI.hs

```diff
@@ -1,16 +1,25 @@
 module Radicle.Internal.CLI
-    ( getConfig
-    , getHistory
+    ( getConfigFile
+    , getHistoryFile
     )
 where
 
 import           Protolude
 import           System.Directory (XdgDirectory(..), getXdgDirectory)
 
 -- | Uses XDG_CONFIG_HOME if available.
-getConfig :: IO FilePath
-getConfig = getXdgDirectory XdgConfig "rad/config.rad"
+-- | Location of the default radicle file to interpretradicle history.
```

Comment:
I think the line above this should be deleted, or the pipe in this one.

Also, this isn't really related to history?
```suggestion
-- | Location of the radicle file to interpret.
```